utils.go

```diff
@@ -1,21 +1,17 @@
 package bux
 
 import (
-	"context"
-	"fmt"
 	"runtime/debug"
 	"strings"
 
-	zLogger "github.com/mrz1836/go-logger"
+	"github.com/rs/zerolog"
 )
 
-func recoverAndLog(ctx context.Context, log zLogger.GormLoggerInterface) {
+func recoverAndLog(log *zerolog.Logger) {
 	if err := recover(); err != nil {
-		log.Error(ctx,
-			fmt.Sprintf(
-				"panic: %v - stack trace: %v", err,
-				strings.ReplaceAll(string(debug.Stack()), "\n", ""),
-			),
+		log.Error().Msgf(
```

Comment:
Is it possible to use `zerolog.Panic` here ?

Reply:
Are you sure we want to create Panic log? it will stop goroutine

Reply:
Ok. Leave log.Error